src/models/user/read.js

```diff
@@ -204,7 +204,12 @@ export const getProfile = async (id, currentIdentity) => {
         LEFT JOIN job_categories j ON j.id=e.job_category_id
         LEFT JOIN experience_credentials ec ON ec.experience_id=e.id
         WHERE e.user_id=u.id AND (ec.status IS NULL OR ec.status != 'ISSUED')
-    ) AS experiences
+    ) AS experiences,
+    (
+	    SELECT COUNT(*)::int
+	    FROM connections c2
+	    WHERE (c2.requested_id=u.id OR c2.requester_id=u.id) AND c2.status='CONNECTED'
+    ) AS connections
```

Comment:
you need to add this for other profile methods also

Reply:
@jeyem I added to all of the similar methods